sound/soc/sof/intel/hda-loader.c

```diff
@@ -277,7 +279,7 @@ static int cl_copy_fw(struct snd_sof_dev *sdev, struct hdac_ext_stream *hext_str
 	}
 
 	status = snd_sof_dsp_read_poll_timeout(sdev, HDA_DSP_BAR,
-					HDA_DSP_SRAM_REG_ROM_STATUS, reg,
+					chip->rom_status_reg, reg,
```

Comment:
Can we also change the trace `"error: %s: timeout HDA_DSP_SRAM_REG_ROM_STATUS read\n"` at line 295?

Reply:
ack, i'll revisit the error handling in this 2018 file.

Reply:
actually no, this is an unrelated change. we want to avoid using the 'error' prefix in new code, but in existing code we leave it as is to avoid churn. This PR is big enough already.

Reply:
> actually no, this is an unrelated change. we want to avoid using the 'error' prefix in new code, but in existing code we leave it as is to avoid churn. This PR is big enough already.

Sorry, @plbossart I didn't mean the 'error' prefix. I mean 'HDA_DSP_SRAM_REG_ROM_STATUS' instead. We are changing the reading from `HDA_DSP_SRAM_REG_ROM_STATUS` to `chip->rom_status_reg` now. The `error: %s: timeout HDA_DSP_SRAM_REG_ROM_STATUS read\n` trace will be misleading when chip->rom_status_regv is not HDA_DSP_SRAM_REG_ROM_STATUS.
What I suggested was
```
		dev_err(sdev->dev,
			"error: %s: timeout rom_status_reg %#x read\n",
			__func__, chip->rom_status_reg);
```